schemas/vulnerability.yml

```diff
@@ -0,0 +1,149 @@
+---
+- name: vulnerability
+  title: Vulnerability
+  group: 2
+  short: Fields to describe the vulnerabilty relevant to an event.
+  description: >
+    The vulnerability fields describe information about a vulnerabilty that is relevant to an event.
+
+  type: group
+  fields:
+
+    - name: classification
+      level: extended
+      type: keyword
+      short: Classification of the vulnerability.
+      description: >
+        The classification of the vulnerability scoring system. For example (ex. https://www.first.org/cvss/)
+
+      example: CVSS
+
+    - name: enumeration
+      level: extended
+      type: keyword
+      short: Enumeration of the vulnerability.
+      description: >
+        The enumeration of the common identifiers for publicaly known vulnerabilities. For example (ex. https://cve.mitre.org/about/)
```

Comment:
I think it's not obvious enough that this is meant to capture the "type of vulnerability ID" (CVE), not the actual ID (CVE-2019-0000).

Perhaps update the description with more explicit wording like the following:

> The type of identifier used for this vulnerability, for example "CVE-2019-0000" would have `vulnerability.enumeration:CVE`

Perhaps the word "enumeration" has this clear meaning for security specialists, but it's not obvious to a mere security fan like me :-)

Reply:
Done.